server/routes/admin.js

```diff
@@ -4,13 +4,9 @@ import AuthorController from '../controllers/AuthorController';
 
 const admin = express.Router();
 const BASE_URL = '/admin';
-const { AuthorValidator } = middlewares;
-// const { verifyToken, authorizeUser, AuthorValidator } = middlewares;
 const { addAuthor } = AuthorController;
-const { authorValidation } = AuthorValidator;
-// const authorizeAdmin = authorizeUser(['admin']);
+const { verifyToken, AuthorValidator: { authorValidation } } = middlewares;
 
-admin.post(`${BASE_URL}/author`, authorValidation(), addAuthor);
-// admin.post(`${BASE_URL}/author`, authorValidation(), verifyToken, authorizeAdmin, addAuthor);
```

Comment:
Let's uncomment this line instead of deleting it. Also, verifyToken should be the first middleware

Reply:
Done